pkg/webhooks/workload_webhook_test.go

```diff
@@ -232,6 +234,48 @@ func TestValidateWorkload(t *testing.T) {
 				field.Invalid(firstPodSetSpecPath.Child("containers").Index(0).Child("resources", "requests").Key(string(corev1.ResourcePods)), nil, ""),
 			},
 		},
+		"should podSetUpdates have the same number of podSets": {
```

Comment:
Add one case for empty podSetUpdates, as that's valid

Reply:
added https://github.com/kubernetes-sigs/kueue/pull/1220/commits/7799023ffef622e04d748162ae003fbf416f5ac9

Reply:
ah, sorry, I missed it.